GitVersionCore.Tests/IntegrationTests/GitFlow/PatchScenarios.cs

```diff
@@ -13,13 +13,13 @@ public void PatchLatestReleaseExample()
             // create hotfix
             fixture.Repository.CreateBranch("hotfix-1.2.1").Checkout();
 
-            fixture.AssertFullSemver("1.2.1-beta.1+1");
+            fixture.AssertFullSemver("1.2.1-beta.1+0");
```

Comment:
Good example of new commit counting. We have not commited to the new branch, so why was it 1?